### ISSUE TITLE
My Sites: increase size of Switch Site button to meet recommended size

### ISSUE THREAD

**Issue by mattmiklic** [wordpress-mobile/WordPress-Android#6091]
The Switch Site button is currently 38dp; it should be increased to 48px to match the Material Design <a href="https://material.io/guidelines/usability/accessibility.html#accessibility-style">recommended touch targets</a>.

<img width="787" alt="screen-shot-2017-06-16-at-3-17-30-pm" src="https://user-images.githubusercontent.com/1396552/27242404-0786127e-52ab-11e7-859f-64f669da9b2d.png">

**Comment by tonyr59h** [comment]
I think we should also make sure the entire lower portion of the header is the area of the touch target. Right now there are areas on the left, bottom, and top that are not included in the touch target that end up resolving as a tap on the top portion of the header (which opens a site preview in a web view).

![](https://cldup.com/607iPJtNW4.png)

cc @mattmiklic
